polls/tool/src/webapp/templates/pollOptionBatch.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html      PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html rsf:id="polls-html" xmlns="http://www.w3.org/1999/xhtml" xmlns:rsf="http://ponder.org.uk/rsf">
+<head>
+<title>Polls: Add/edit options in batch</title>
+<link  rsf:id="scr=portal-matter" href="../css/tool_base.css" type="text/css" rel="stylesheet" media="all"/>
+<link rsf:id="scr=portal-matter" href="../css/default/tool.css" type="text/css" rel="stylesheet" media="all"  />
+<script rsf:id="scr=portal-matter" type="text/javascript" language="JavaScript" src="/library/js/headscripts.js"></script>
+</head>
+
+<body rsf:id="scr=sakai-body">
+<div class="portletBody">
+    <h3 rsf:id="new-option-batch-title">Add/Edit Options in batch</h3>
```

Comment:
Page headings should use the new SWITCH format of:

`<div class="page-header"><h1>Page Title</h1></div>`

instead of an h3.

Reply:
Good